allensdk/internal/brain_observatory/roi_filter_utils.py

```diff
@@ -286,6 +291,8 @@ def get_indices_by_distance(object_list_points, mask_points):
     Require a distance of 0 (perfect match) and a unique match between
     masks and object_list entries.
     '''
+    if np.array(mask_points).ndim != 2:
+        raise ValueError("number of dimensions is incorrect.")
```

Comment:
Do you mind adding the actual value of `mask_points.ndim` to the message? That might help in debugging.

Reply:
done.
Though, this will never actually get invoked anymore because of the earlier catch of "no ROIs found"
But, it is good to have here in case someone re-uses this function.
Also, we should have been able to just check against the scipy exception, but their wording changed across versions, so I added this.